source-map.bs

```diff
@@ -2,16 +2,21 @@
 Title: Source Map
 H1: Source Map
 Shortname: source-map
-Level: 1
+Level: none
 Status: STAGE0
 URL: https://tc39.es/source-map/
+Repository: https://github.com/tc39/source-map/
+Issue Tracking: GitHub Issues https://github.com/tc39/source-map/issues
+Test Suite: https://github.com/tc39/source-map-tests
+Editor Term: Editor, Editors
 Editor: Asumu Takikawa, Igalia
 Former Editor: Victor Porof, Google
 Former Editor: John Lenz, Google
 Former Editor: Nick Fitzgerald, Mozilla
 Previous Version: https://docs.google.com/document/d/1U1RGAehQwRypUTovF1KRlpiOFze0b-_2gc6fAH0KY0k/edit?pli=1#
 Repository: source-map/source-map-spec
-Abstract: A specification for mapping transpiled source code back to the original sources.  This specification is a living document and describes a hardened version of the Source Map v3 specification.
+Abstract: This Ecma standard defines the Source Map format, used for mapping transpiled source code back to the original sources.
```

Comment:
Curious if we'll get any pushback on this.

Reply:
On what part specifically? :)

Reply:
Oh well, that was already there in the abstract :P

Reply:
🚢 !